src/utils.ts

```diff
@@ -478,6 +479,95 @@ export async function handleExternalProject(
   }
 }
 
+/**
+ * Config External CMake Project config file as an IoT Workbench Container
+ * Project. Throw cancel operation error if not CMake project. Update project
+ * host type and board id in IoT Workbench project file.
+ * @param scaffoldType
+ */
+export async function configExternalCMakeProjectToIoTContainerProject(
+    scaffoldType: ScaffoldType): Promise<void> {
+  const projectRootPath = getFirstWorkspaceFolderPath();
+  // Check if cmake project
+  const cmakeFile = path.join(projectRootPath, FileNames.cmakeFileName);
+  if (!await FileUtility.fileExists(scaffoldType, cmakeFile)) {
+    const message = `Missing ${
+        FileNames.cmakeFileName} to be configured as Embedded Linux project.`;
+    vscode.window.showWarningMessage(message);
+    throw new CancelOperationError(message);
+  }
+
+  const iotworkbenchprojectFile =
```

Comment:
can we make "iotworkbenchproject" as camel case to better read?

Reply:
The former "iotworkbenchProjectFile" uses this case because it wants to represent the "**`.iotworkbenchproject` file**", in which case it is camel case. Since we have two kinds of reperesentation "*iotworkbenchprojectFile*" and "*iotWorkbenchProjectFile*", I will delete the former one and use "*iotWorkbenchProjectFile*" for consolidation.

Reply:
Fixed.